packages/keyring-controller/src/KeyringController.ts

```diff
@@ -125,20 +120,22 @@ export type KeyringControllerOptions = {
   setSelectedAddress: PreferencesController['setSelectedAddress'];
   setAccountLabel?: PreferencesController['setAccountLabel'];
   encryptor?: any;
-  keyringBuilders?: any[];
+  keyringBuilders?:
+    | { (): Keyring<Json>; type: string }
+    | ConcatArray<{ (): Keyring<Json>; type: string }>;
```

Comment:
Nit: This type would be simpler:

```suggestion
  keyringBuilders?: { (): Keyring<Json>; type: string }[];
```

It should be compatible with the more-complex type currently in the internal keyring controller as well